From b36e09f5cd6f4c894c58e5c6b2ec92c72b75bdbd Mon Sep 17 00:00:00 2001
From: Justin Jacobs <jajdorkster@gmail.com>
Date: Sat, 5 Apr 2025 18:27:55 -0400
Subject: [PATCH] Minimum CMake version = 3.5

From the CMake 4.0 release notes:
> Compatibility with versions of CMake older than 3.5 has been removed.
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 897c19c3..def8121c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,5 +1,5 @@
 Project (Flare)
-cmake_minimum_required (VERSION 2.8.12)
+cmake_minimum_required (VERSION 3.5)
 
 Set (PACKAGE "FLARE")
 Set (VERSION "1.14")
-- 
2.49.0

